wordpress-mobile/WordPress-Android
# Feature/improve strings lane

This PR follows up to https://github.com/wordpress-mobile/WordPress-Android/pull/9078 by adding a local action that takes care of:
- Checking that the repository is clean at start.
- Committing the modified files.

To test:
1. Run `bundle exec fastlane update_ps_strings version:11.6` and verify that nothing changes in the .po file.
2. Check out a new branch from this one.
3. Make some changes to one of the .txt files and commit them.
4. Run `bundle exec fastlane update_ps_strings version:11.7` and verify that the .po file has been correctly updated. (Note that the lane may fail on the last git push command if the branch you are using doesn't have an upstream set).

Update release notes:
